### DISCUSSION TITLE
miss deployment on maven ical4j-4.0.0-beta1-pre

### DISCUSSION THREAD

**Discussion by leccyril** [ical4j/ical4j#577] [General]
Hi, you forget to deploy this version on maven repository, is it possible ?

ical4j-4.0.0-beta1-pre

thank you !

**Comment by benfortuna** [comment]
Hello, this is actually a marker tag for the next version, but it hasn't been released yet. I hope to release maybe in a week or two.

If you need to urgently test something you can try the snapshot build in the meantime:

https://oss.sonatype.org/content/repositories/snapshots/org/mnode/ical4j/ical4j/4.0.0-beta1-SNAPSHOT/

**Reply by benfortuna** [reply]
Note that ical4j beta1 is now available. https://search.maven.org/artifact/org.mnode.ical4j/ical4j
